### DISCUSSION TITLE
how can protect dashboard with username and password

### DISCUSSION THREAD

**Discussion by engahmedtahaa** [jobrunr/jobrunr#203] [Q&A] [answered]
we have a requirement form security to protect the dashboard by username and password, anyone have any idea

**Comment by tan9** [comment, marked as answer]
https://github.com/jobrunr/jobrunr/blob/master/jobrunr-spring-boot-starter/src/main/java/org/jobrunr/autoconfigure/JobRunrProperties.java#L165

These exists `org.jobrunr.dashboard.username` and  `org.jobrunr.dashboard.password` Spring Boot application properties.

**Comment by engahmedtahaa** [comment]
thanks a lot, is there is anyway to add the password encrypted

**Comment by rdehuyss** [comment]
FYI - JobRunr Pro now supports OpenId authentication.
